code/web/src/modules/user/Profile.js

```diff
@@ -104,27 +130,112 @@ class Profile extends PureComponent {
               marginLeft: '2em',
               padding: '2em',
               textAlign: 'center',
-              objectFit: 'contain',
               maxWidth: 250,
-              maxHeight: 250
+              maxHeight: 250,
             }}>
-            <ImageTile height={250} width={250} image={userImage} shadow={level3}/>
+            <ImageTile
+              style={{backgroundRepeat: 'no-repeat', backgroundSize: 'cover', backgroundPosition: 'center'}}
```

Comment:
ahhhh this makes sense, working with JSX styled components is a bit more work than anticipated!